molecule/templates/molecule.yml.j2

```diff
@@ -107,7 +107,7 @@ vagrant:
 #        cpus: 2
 
   instances:
-    - name: vagrant-01
+    - name: {{ role }}-01
```

Comment:
For the template it's prob not a bad idea, since it can be changed by the end-user.  It's just a default.